e2e/generateZkpKeysFromMnemonic.test.ts

```diff
@@ -0,0 +1,23 @@
+import { Client } from "../libs/client";
+import testVars from "./config";
+
+describe("Verify client API get contract address", () => {
```

Comment:
- This is not a test for get contract address, but a test for zkp keys
- This is not and e2e test, but an integration test
- There was already an [integration test for zkp keys](https://github.com/maticnetwork/nightfall-sdk/blob/master/__tests__/integration/nightfall/keys.test.ts) here in the sdk